env.example.yml

```diff
@@ -2,4 +2,6 @@ LAMBDA_EXEC_SG: Insert AWS Security Group ID Here (it must be in the same VPC as
 LAMBDA_EXEC_SUBNET: Insert AWS Subnet ID Here (it must be in the same VPC as the security group)
 BUGSNAG_NOTIFIER_KEY: INSERT BUGSNAG NOTIFIER KEY HERE
 GEOCODERS: <Stringified JSON Array of OTP-UI `GeocoderConfig`s>
-BACKUP_GEOCODERS: <Stringified JSON Array of OTP-UI `GeocoderConfig`'s. Same length and order as GEOCODERS> 
+BACKUP_GEOCODERS: <Stringified JSON Array of OTP-UI `GeocoderConfig`'s. Same length and order as GEOCODERS>
+
+LOCATION_COMPARISON_COORDINATE_PRECISION: defaults to 4 (~10m). What precision to use when comparing if two locations are the same
```

Comment:
Could you rename to `COORDINATE_COMPARISON_PRECISION_DIGITS`? (Emphasis on "digits")